lib/spack/docs/chain.rst

```diff
@@ -11,7 +11,8 @@ Chaining Spack Installations
 
 You can point your Spack installation to another installation to use any
 packages that are installed there. To register the other Spack instance,
-you can add it as an entry to ``upstreams.yaml``:
+you can add it as an entry to ``path/to/spack/etc/spack/upstreams.yaml``,
+if ```upstreams.yaml``` is not there, crete it similar to:
```

Comment:
This isn't the only location where `upstreams.yaml` can be located. Any of the configuration scopes at https://spack.readthedocs.io/en/latest/configuration.html#configuration-scopes can support this. How about this?
```suggestion
you can add it as an entry to ``upstreams.yaml`` at any of the
:ref:`configuration-scopes`:
```
This will look similar to the end of the introduction section at https://spack.readthedocs.io/en/latest/packages_yaml.html.